setup.py

```diff
@@ -51,5 +51,8 @@ def extract_version(filename: str):
       url="https://github.com/UC-Davis-molecular-computing/scadnano-python-package",
       long_description=long_description,
       long_description_content_type='text/markdown; variant=GFM',
-      requires=['xlwt']
-      )
+      requires=['xlwt'],

```

Comment:
I just want to make absolutely sure that this will still allow the package to be installed on Python 3.7 or higher. The syntax looks like it's saying that we require Python 3.6 or below.

But is this just saying that, *if* the Python version is 3.6 or below, *then* install dataclasses?

Reply:
Yes I tested it. Simple way to test it create two virtual env, one under python 3.6 and one under python 3.7 and theninside each venv, run "python setup.py", the installation is successful in both cases.

Reply:
And, in the venv with python 3.6, the dataclass package is present and importable after running the script.

Reply:
The syntax is indeed saying, care about the previous package iff this condition on python version is met